src/utils/dates.js

```diff
@@ -10,3 +10,9 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+export function getDaysBetweenDates(previousPurchaseDate) {
```

Comment:
i would consider what to name this function - if i wanted to get the days between dates, i would expect this function to accept two values, date1 and date 2. Since you are only passing in one date and compare it to today, i would name this function something to reflect that. 

This is so when this util function is needed for something else in the future it's easy to tell what it can do.